data-formats/src/ownershipvoucher.rs

```diff
@@ -318,6 +326,10 @@ impl OwnershipVoucher {
             .get(OwnershipVoucherIndex::Header as usize)
             .unwrap()
     }
+
+    pub fn contents(&self) -> &ParsedArray<ParsedArraySize5> {
```

Comment:
This is an internal detail that was very explicitly do not want public.
It has happened in the past that the OV format changes (size4 to size5 for example), and the whole purpose of this file is to abstract that away for other code.